.github/ISSUE_TEMPLATE/enhancement.yaml

```diff
@@ -0,0 +1,22 @@
+name: Enhancement Tracking Issue
+description: Provide supporting details for a feature in development
+labels: kind/feature
+body:
+  - type: textarea
+    id: feature
+    attributes:
+      label: What would you like to be added?
+      placeholder: |
+        NOTE: Feature requests are unlikely to make progress as an issue.  Instead, please consider engaging with SIGs on slack and mailing lists.
+
+        A proposal that works through the design along with the implications of the change can be opened as a KEP.
+        See https://git.k8s.io/enhancements/keps#kubernetes-enhancement-proposals-keps
```

Comment:
Not necessarily a blocker, but no one can copy paste from placeholder text. We may want to either add a [description](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-1) with the link, similar to the [Netlify site request form](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-integration&template=site-request.yml&title=REQUEST%3A+Netlify+site+for+%3Crepo%3E) or create a shortlink like go.k8s.io/keps that could be used as an easy to type shortcut that could be used in the placeholder text.

Reply:
Will try as description